pkg/actors/internal/reminders/storage/scheduler/scheduler.go

```diff
@@ -203,6 +205,11 @@ func (s *scheduler) Get(ctx context.Context, req *api.GetReminderRequest) (*api.
 			"jobType":   "reminder",
 		}
 		log.Errorf("Error getting reminder job %s due to: %s", req.Name, err)
```

Comment:
Should it still log an error? Or it good be moved to a debug?

Reply:
Debug makes sense IMO